### PR TITLE
Add schedule and order classes

### PR DESCRIPTION
**Pull request by qiujiaaa**

### CODE REVIEW

File: src/main/java/seedu/address/model/order/Id.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.order;
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/model/order/Id.java]: Will need to ensure this class is generating unique keys right?

File: src/main/java/seedu/address/model/schedule/DateTime.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+     * @param dateTime A valid Date object.
+     */
+    public DateTime(Date dateTime) {
+        requireNonNull(dateTime);
+        checkArgument(isValidDateTime(dateTime.toString()), MESSAGE_CONSTRAINTS);
+        this.dateTime = dateTime;
+    }
+
+    /**
+     * Returns true if a given string is a valid order ID.
```
Comment (qiujiaaa) [inline src/main/java/seedu/address/model/schedule/DateTime.java]: javadoc

File: src/main/java/seedu/address/model/order/Order.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
+package seedu.address.model.order;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/model/order/Order.java]: Good job on the use of UUID!

File: src/main/java/seedu/address/model/schedule/Schedule.java [hunk 4]
```diff
@@ -0,0 +1,112 @@
+package seedu.address.model.schedule;
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/model/schedule/Schedule.java]: For this class, we can deal with the schedule update delete later on in the project? can leave like that for now

File: src/main/java/seedu/address/model/order/Status.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.order;
+
+/**
+ * Represents an Order's status in the SML.
+ * Guarantees: Only 3 valid status.
+ */
+public enum Status {
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/model/order/Status.java]: We will need to add cancel

**Pull request closed without merging**